qemu/tests/boot_N_1_virtserialports.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_test
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot guest with multiple virtio-serial-devices
+
+    1. Boot a guest with 26 virtio-serial-pcis
+    2. Every bus is used for one virtserialport
+    3. Transfer data via evey port
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    num_bus = params.get_numeric("numberic_bus")
+    for i in range(2, num_bus + 1):
+        serial_name = 'vs%d' % i
```

Comment:
Why use `numberic_bus = 26` ?   Considering the default `vs1`, 26 serials in total, from the test case, it is 28 ?

Reply:
some others' PCI devices will take some buses so will case that there is not enough free bus so that VM can't boot successfully when numberic_bus is over 26.  And I have confirmed with xiagao it's ok.

Reply:
Got it, and make sense, thanks!